bliss/models/binary.py

```diff
@@ -218,17 +220,16 @@ def make_plots(self, batch, n_samples=16):
         nrows = int(n_samples ** 0.5)  # for figure
 
         # extract non-params entries so that 'get_full_params' to works.
-        exclude = {"images", "background"}
+        exclude = {"images", "background", "loss", "acc"}
         true_tile_params = TileCatalog(
             self.tile_slen, {k: v for k, v in batch.items() if k not in exclude}
         )
         true_params = true_tile_params.to_full_params()
         # prediction
-        pred = self.get_prediction(batch)
         tile_est = true_tile_params.copy()
-        tile_est["galaxy_bools"] = pred["galaxy_bools"]
-        tile_est["star_bools"] = pred["star_bools"]
-        tile_est["galaxy_probs"] = pred["galaxy_probs"]
+        tile_est["galaxy_bools"] = batch["galaxy_bools"]
+        tile_est["star_bools"] = batch["star_bools"]
+        tile_est["galaxy_probs"] = batch["galaxy_probs"]
```

Comment:
sorry not sure I follow why this change?

Reply:
I was running out of memory on binary at validation time. This is because it was running the prediction for all validation batches at once. I changed it to use the output of each `validation_step` instead; it should be exactly the same result in the end.